docs/source/community.rst

```diff
@@ -63,17 +63,27 @@ All code contributed via pull request needs to adhere to the following guideline
 
 .. tip::
 
-    During continuous integration, the code is checked automatically with `Flake8`_.
-    Run the following commands to set up a pre-commit hook that will ensure your code is compliant before committing:
+    During continuous integration, the code and the documentation is checked automatically using `Flake8`_,`Pydocstyle`_, and `Mypy`_.
+    Run the following commands to set up a pre-commit hook, using `Pre-commit`_, that will ensure your code and documentation are compliant before committing:
 
     .. code-block:: bash
 
-        flake8 --install-hook git
-        git config --bool flake8.strict true
+        pre-commit install
 
+    To install and run `Pre-commit`_ for all the files present in the repository, run the following commands:
```

Comment:
Is this something that you run on each commit or you run once on the project and it's always in place? This point could be more clear here.

Reply:
I addressed your comment. Please review the changes.